### PR TITLE
Fix: Nested Elements - JS error thrown a nested item is clicked when in... [ED-9495]

### PR DESCRIPTION
**Pull request by udidol**
…place editing different document [ED-9495]

## PR Type
What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Fix: Mega Menu - JS error thrown a mega menu item is clicked when in-place editing different document

[ED-9495]: https://elementor.atlassian.net/browse/ED-9495?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js [hunk 1]
```diff
@@ -286,7 +286,7 @@ export default class NestedTabs extends Base {
 	changeActiveTab( tabIndex, fromUser = false ) {
 		// `document/repeater/select` is used only in edit mod, and only when its not internal call,
 		// in other words only in editor and when user triggered the change.
```
Comment (udidol) [inline modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:288]:
```suggestion
		// `document/repeater/select` is used only in the editor, only when the element
		// is in the currently-edited document, and only when its not internal call,
```

**Review by davseve — COMMENTED** [review]
Do we need tests or a mindmap?

**Comment by udidol** [comment]
@davseve There is a mind map, no tests because this is a JS error, there are no functionality or UI issues to test

**Comment by udidol** [comment]
@davseve There is a mind map, but no automated tests, because this fixes just a JS error, there are no functionality or UI issues caused by this problem.

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**